plugin/src/main/groovy/grails/plugin/springsecurity/acl/SpringSecurityAclGrailsPlugin.groovy

```diff
@@ -173,14 +173,13 @@ class SpringSecurityAclGrailsPlugin extends Plugin {
 		objectIdentityRetrievalStrategy(GormObjectIdentityRetrievalStrategy)
 
 		// acl cache
-		aclCacheManager(EhCacheManagerFactoryBean) {
-			cacheManagerName = 'spring-security-acl-cache-' + UUID.randomUUID()
-		}
-		ehcacheAclCache(EhCacheFactoryBean) {
+		aclCacheManager(JCacheCacheManager)
```

Comment:
It makes sense to use the spring cache, but we probably should document this change since it has different defaults.

Reply:
I opened an issue (#46) to update the docs for Grails 7.